plugin/src/main/java/org/wildfly/plugin/provision/AbstractProvisionServerMojo.java

```diff
@@ -265,4 +282,28 @@ static Path resolvePath(MavenProject project, Path path) {
         return path;
     }
 
+    public void setFeaturePacksFromProperty() {
+        String fps = System.getProperty(PropertyNames.WILDFLY_PROVISIONING_FEATURE_PACKS);
```

Comment:
How would this be used? I don't think it's wrong or anything, I'm just trying to understand the use-case.

Reply:
@jamezp , it would be used in this kind of context: https://github.com/wildfly/wildfly-proposals/pull/461/files#diff-81512f32e1f2215e55b0a246a066be2a621704c1d3932621c5963d312b3a8c0dR208

Reply:
You could add a `FeaturePack.set(String)` method which would allow it to be set via a property. I suppose in some ways this is the same, but it's just not the standard.

Reply:
You'd just add something like:
```java
    public void set(final String value) {
        setLocation(value);
    }
```

IIRC Maven will handle the comma split for you.

Reply:
That makes sense. Do you want this merged for now and to it later or would you rather implement that now?

Reply:
@jamezp , I will implement it now.

Reply:
@jamezp , the feature-packs are now set in the standard maven way.